README.md

```diff
@@ -940,7 +941,7 @@ inputs:
     value: 'b'
 ```
 
-Then you can use `abc templates golden-test` to record or verify the tests.
+Then you can use `abc templates golden-test` to record (capture the anticipated outcome akin to expected output in unit test)or verify the tests.
```

Comment:
I still see this line as being 145 characters long

Reply:
done